datamodel/referencedata.d/documenttypecodes.csv

```diff
@@ -0,0 +1,6 @@
+shipment_information_type_code,shipment_information_type_name
```

Comment:
Will this work - or does it just ignore the column names? In the table they are called document_type_code

Reply:
It ignores I think but either way it's better to correct it. Good spot there, missed it.